development/kops/cluster_wait.sh

```diff
@@ -45,15 +45,10 @@ function k {
 echo 'Waiting for cluster to come up...'
 k apply -f ./${KOPS_CLUSTER_NAME}/aws-iam-authenticator.yaml
 
-PATCH='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--secure-port=4443" },{"op": "replace", "path": "/spec/template/spec/containers/0/ports/0/containerPort", "value": 4443 }]'
```

Comment:
This is now the default port in kops since they bumped to the newer metrics server